transaction/core/tests/digest-test-vectors.rs

```diff
@@ -19,13 +19,16 @@ fn test_origin_tx_outs() -> Vec<TxOut> {
     accounts
         .iter()
         .map(|acct| {
-            TxOut::new(
+            let mut tx_out = TxOut::new(
                 rng.next_u32() as u64,
                 &acct.default_subaddress(),
                 &RistrettoPrivate::from_random(&mut rng),
                 EncryptedFogHint::fake_onetime_hint(&mut rng),
+                Default::default(),
             )
-            .unwrap()
+            .unwrap();
+            tx_out.e_memo.clear();
```

Comment:
Whats the idea here?

Reply:
i was trying to avoid changing the existing test vectors, since there isn't going to be a new origin block

when you do TxOut::new( ..., MemoPayload::default()), you still get an encrypted memo that's 34 bytes long, but it's an encryption of zeroes

if we do `e_memo.clear()` then there are actually no bytes in the e_memo as in the real origin block

i can add a code comment, i added a code comment along these lines somewhere else also